src/taco-cli/cli/create.ts

```diff
@@ -78,23 +78,23 @@ class Create extends commands.TacoCommandBase {
             return Q.reject<ICommandTelemetryProperties>(err);
         }
 
-        var self = this;
+        var self: Create = this;
         var templateDisplayName: string;
 
         return this.createProject()
-            .then(function (templateUsed: string): Q.Promise<any> {
+            .then(function(templateUsed: string): Q.Promise<any> {
                 templateDisplayName = templateUsed;
 
-                var kitProject = self.isKitProject();
+                var kitProject: boolean = self.isKitProject();
                 var valueToSerialize: string = kitProject ? self.commandParameters.data.options["kit"] : self.commandParameters.data.options["cordova"];
 
                 return projectHelper.createTacoJsonFile(self.commandParameters.cordovaParameters.projectPath, kitProject, valueToSerialize);
             })
-            .then(function (): Q.Promise<any> {
```

Comment:
Looks like you the space between "function" and "()" has been deleted in some files but not in many other files. Lets be consistent here.